apps/server/src/modules/user/service/user.service.ts

```diff
@@ -60,7 +60,7 @@ export class UserService {
 
 	async getResolvedUser(userId: EntityId): Promise<OauthCurrentUser> {
 		const user: UserDO = await this.findById(userId);
-		const account: AccountDto = await this.accountService.findByUserIdOrFail(userId);
+		const account: Account = await this.accountService.findByUserIdOrFail(userId);
```

Comment:
no need to specify type, which is infered

Reply:
We can open this discussion again and again...